main/src/ca/uwaterloo/flix/language/ast/KindedAst.scala

```diff
@@ -178,13 +178,15 @@ object KindedAst {
 
     case class InvokeStaticMethod2(clazz: Class[_], methodName: Name.Ident, exps: List[Expr], mvar: Type.Var, tvar: Type.Var, evar: Type.Var, loc: SourceLocation) extends Expr
 
+    case class GetField2(exp: Expr, fieldName: Name.Ident, mvar: Type.Var, tvar: Type.Var, evar: Type.Var, loc: SourceLocation) extends Expr
```

Comment:
mvar was for method var, so I suppose fvar would be better.

We could also call all of them jvar.